test/input/illegal/declarations.sim

```diff
@@ -5,21 +5,21 @@ end
 extern t : int;
 extern V : set{E}(t);
 
-%%% bad-lattice-link-set-type-1
+%%% bad-grid-link-set-type-1
 element E
 end
 
 extern Q : set{E};
 extern P : set{E}(Q);
-extern S : lattice[1]{E}(P);
+extern S : grid{E}[1](P);
```

Comment:
I'm really sorry about the back and forth(!!), but now I'm thinking maybe your original `grid[1]{E}(P)` is better, since the dimension refers to the `grid` and both `E` and `P`. What do you think?

Depending on whether you think this this makes more sense, I'm happy to make the change if you'd like since I'm the one causing the back and forth.